<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QLineEdit" name="pg_port_line_edit">
     <property name="placeholderText">
      <string>[Leave empty to use standard port 5432]</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="pg_host_label">
     <property name="text">
      <string>Host</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QCheckBox" name="pg_use_super_login">
     <property name="text">
      <string>Use logins from settings (superuser) for schema generation</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="pg_service_combo_box"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="pg_schema_label">
     <property name="text">
      <string>Schema</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="pg_port_label">
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Service</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="pg_ssl_mode_combo_box">
     <property name="toolTip">
      <string>This option determines whether or with what priority a secure SSL TCP/IP connection will be negotiated with the server.</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="pg_schema_combo_box"/>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QgsAuthSettingsWidget" name="pg_auth_settings" native="true"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="pg_database_label">
     <property name="text">
      <string>Database</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="pg_host_line_edit">
     <property name="text">
      <string>localhost</string>
     </property>
     <property name="placeholderText">
      <string>Database Hostname</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="pg_database_line_edit">
     <property name="placeholderText">
      <string>Database Name</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="pg_ssl_mode_label">
     <property name="toolTip">
      <string>This option determines whether or with what priority a secure SSL TCP/IP connection will be negotiated with the server.</string>
     </property>
     <property name="text">
      <string>SSL Mode</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsAuthSettingsWidget</class>
   <extends>QWidget</extends>
   <header>qgis.gui</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pg_host_line_edit</tabstop>
  <tabstop>pg_port_line_edit</tabstop>
  <tabstop>pg_database_line_edit</tabstop>
  <tabstop>pg_schema_combo_box</tabstop>
  <tabstop>pg_use_super_login</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
